src/editor.rs

```diff
@@ -7,6 +7,8 @@ use crate::Terminal;
 use std::io::Error;
 use termion::color;
 use termion::event::Key;
+use clipboard::ClipboardContext;
+use clipboard::ClipboardProvider;
```

Comment:
The `use` statements should also be in lexicographical order! You can run `cargo fmt` to handle this automatically.